CONTRIBUTING.md

```diff
@@ -14,14 +14,14 @@ We use a bunch of nice Open Source tools to make collaboration of both developer
 
 - We use [Node.js](https://github.com/nodejs/node) as JavaScript run-time environment to build the page.
 - All contents are written in clean and sexy Markdown, see [Markdown (Wikipedia.org)](https://en.wikipedia.org/wiki/Markdown).
-    - We use the [markdown-it](https://github.com/markdown-it/markdown-it) parser to have some additional parsing features available.
-    - A table of content can be inserted in any README.md via `[[_TOC_]]` placeholder.
+  - We use the [markdown-it](https://github.com/markdown-it/markdown-it) parser to have some additional parsing features available.
```

Comment:
```suggestion
  - We use the [markdown-it](https://markdown-it.github.io/) parser to have some additional parsing features available.
```
As with Node.js, it might make sense to link to the website instead of the repository.